src/about.html

```diff
@@ -1,16 +1,4 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Healthy Bites</title>
-    <link rel="stylesheet" href="./css/style.css">
-</head>
-<body>
-    <header class="divider" id="main-header"></header>
 
+    <header class="divider" id="main-header"></header>
     <main>This is About Page</main>
-    <footer id="main-footer"></footer>
-    <script type="module" src="js/main.js"></script>
-</body>
-</html>
```

Comment:
@JosueBCe You had already done this for this file so you were on the right track, you just needed to do it for everything else as well.